server/src/main/java/org/spine3/server/stand/AggregateStateId.java

```diff
@@ -37,19 +51,33 @@
  * @author Alex Tymchenko
  */
 public final class AggregateStateId<I> {
+
+    private static final AggregateStateIdStringifier stringifier =
+            new AggregateStateIdStringifier();
+
+    static {
+        StringifierRegistry.getInstance()
```

Comment:
This way of registration is dangerous. Since it's a `static` initialization block, it may never be executed. I would put it somewhere else.